job/job.go

```diff
@@ -34,6 +35,34 @@ type Job struct {
 	instance *packet.Instance
 }
 
+var tinkWorkflowClient = client.GetWorkflowsFromTink
+
+// discoverWorkflowForMac fetches the workflow for a given MAC
+// hwID is the hardware/worker ID corresponding to the MAC
+func discoverActiveWorkflowForMac(hwID string) (bool, error) {
+	var active bool = false
+
+	s := fmt.Sprintf("fetching workflows for hardware %s", hwID)
+	joblog.With("dhcp", hwID).Info(s)
+
+	wcl, err := tinkWorkflowClient(hwID)
+	if err != nil {
+		return false, err
+	}
+
+	// wcl is workflowContextList, for more details refer to workflow.pb.go in tink
```

Comment:
this comment seems unnecessary, any editor should be able to provide this same info

Reply:
done

Reply:
done